setup.py

```diff
@@ -72,22 +83,38 @@ def get_args(cls, dist, header=None):
 setup(
     name='iocage',
     license='BSD',
-    version='0.2.12',
+    version='0.3.0',
```

Comment:
shouldn't this be the only place for our version number

Reply:
Could you create an Issue for that? This should be handled in a different PR.